single-artist.php

```diff
@@ -69,7 +69,15 @@
           ?>
           <div class="masonry-item">
             <?php
-              echo wp_get_attachment_image($image['image_id'], '1920', false, array('class'=>'carousel-trigger','data-no-lazysizes'=>'true','data-index'=>$index));
+              if (!empty($image['image_id'])) {
+                echo wp_get_attachment_image($image['image_id'], '1920', false, array('class'=>'carousel-trigger','data-no-lazysizes'=>'true','data-index'=>$index));
+              } elseif (!empty($image['vimeo_id'])) {
+            ?>
+
+            <div style="padding:56.25% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/<?php echo $image['vimeo_id']; ?>?color=ffffff&title=0&byline=0&portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
```

Comment:
This could have a utility class that does it and its child iframe's styles rather than inlining twice on 2 templates. In the styl boiler we have `.u-video-embed-container` which does this already: https://github.com/interglobalvision/globie-styl-library/blob/master/helpers.styl#L58

Reply:
ok will do